src/wrapper/Makefile.am

```diff
@@ -1,13 +1,18 @@
 lib_LTLIBRARIES = libdyad_sync.la
 libdyad_sync_la_SOURCES = wrapper.c
+libdyad_sync_la_LDFLAGS = $(AM_LDFLAGS) -module -avoid-version -no-undefined -shared
```

Comment:
Should `-module` be added here? We haven't built `libdyad_sync.so` as a module before.

The other flags all make sense. Just a bit concerned about `-module`.

Reply:
I think that would be better with a name change to `dyad_wrap(per)` instead of `libdyad_sync` because it is not supposed to be linked. What do think?

Reply:
we will address in the followup PR.